test/redis/store/marshalling_test.rb

```diff
@@ -23,6 +23,12 @@ def teardown
     @store.get("rabbit").must_equal(@white_rabbit)
   end
 
+  it "marshals on multi set" do
+    @store.mset "rabbit", @white_rabbit, "rabbit2", @rabbit
```

Comment:
We're using parenthesis around method arguments throughout this file, could you please change your code to make it look like everything else?